modules/sdk-coin-ada/src/lib/utils.ts

```diff
@@ -1,8 +1,57 @@
-import { BaseUtils } from '@bitgo/sdk-core';
-import { Ed25519Signature, Address } from '@emurgo/cardano-serialization-lib-nodejs';
+import { AddressFormat, BaseUtils } from '@bitgo/sdk-core';
+import {
+  Address,
+  BaseAddress,
+  Bip32PublicKey,
+  Ed25519Signature,
+  NetworkInfo,
+  StakeCredential,
+} from '@emurgo/cardano-serialization-lib-nodejs';
 import { KeyPair } from './keyPair';
 
 export class Utils implements BaseUtils {
+  createBaseAddressWithStakeAndPaymentKey(
+    stakeKeyPair: KeyPair,
+    paymentKeyPair: KeyPair,
+    network: AddressFormat
+  ): string {
+    let baseAddr;
+    if (network === AddressFormat.mainnet) {
+      // 1. create stake pubKey
+      const key = stakeKeyPair.getKeys().pub;
+
+      const stakePub = Bip32PublicKey.from_bytes(Buffer.from(key));
```

Comment:
why use bip32 key?

Reply:
When I do the same process, PublicKey.from_bytes(key) it throws an invalid size error

Reply:
try Buffer.from(key, 'hex') as key is in hex format